src/test/java/com/gdschongik/gdsc/domain/member/application/OnboardingMemberServiceTest.java

```diff
@@ -38,47 +38,16 @@ private void verifyEmail() {
         memberRepository.save(member);
     }
 
-    @Nested
-    class 가입신청_수행시 {
-
-        @Test
-        void 재학생_인증을_완료했다면_성공한다() {
-            // given
-            setFixture();
-            logoutAndReloginAs(1L, MemberRole.GUEST);
-            verifyEmail();
-
-            // when
-            onboardingMemberService.signupMember(SIGNUP_REQUEST);
-
-            // then
-            Member signupMember = memberRepository.findById(1L).get();
-            assertThat(signupMember.isApplied()).isTrue();
-        }
-
-        @Test
-        void 재학생_인증을_미완료했다면_실패한다() {
-            // given
-            setFixture();
-            logoutAndReloginAs(1L, MemberRole.GUEST);
-
-            // when & then
-            assertThatThrownBy(() -> onboardingMemberService.signupMember(SIGNUP_REQUEST))
-                    .isInstanceOf(CustomException.class)
-                    .hasMessage(ErrorCode.UNIV_NOT_VERIFIED.getMessage());
-        }
-    }
-
     @Nested
     class 회원정보_조회시 {
 
         @Test
-        void 가입신청을_완료헀다면_성공한다() {
+        void 기본회원정보_작성을_완료헀다면_성공한다() {
```

Comment:
```suggestion
        void 기본_회원정보_작성을_완료헀다면_성공한다() {
```

이 파일과 MemberRepositoryTest에 5~6개 정도 있는 것 같아요.
같이 확인부탁드려요.
'기본회원정보'로 찾아서 처리하시면 편할 것 같네요.

Reply:
헉 저 이게 push가 안된것 같아용 왜지..?? 저 분명 로컬에는 다 기본회원정보로 되어있어서 ㅎㅎ..

Reply:
<img width="754" alt="image" src="https://github.com/GDSC-Hongik/gdsc-server/assets/99820610/bc699412-8f14-4309-9774-ae706d67e37b">

안된것 같아서 다시 add해보니까 nothing new이런식으로 뜨는데 혹시 무슨 오륜지 아실까요?

Reply:
리뷰 내용 다시 확인해주세요.

Reply:
아 저 부분 _에 대해서는 수정했는데 제가 _말고는 다 제대로 하고 푸쉬했는데 커밋이 안날린 문제에 대해서 말씀드린거에용!
그 pr리뷰는 이미 반영한 상태입니다~